Apps/W1/EDocumentsConnector/app/src/Logiq/LogiqAuth.Codeunit.al

```diff
@@ -0,0 +1,182 @@
+namespace Microsoft.EServices.EDocumentConnector.Logiq;
+
+codeunit 6380 "Logiq Auth"
```

Comment:
Please move the code to [EDocumentConnectors](https://github.com/microsoft/ALAppExtensions/tree/main/Apps/W1/EDocumentConnectors)

Create app for the connector.
Create app for the test automation.
Feel free to look at Avalara as the example.

In the mean time we will start looking at the code.

Reply:
Created connector and test apps. Moved to EDocumentConnectors folder